modules/cmake_configure_file/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/wep21/cmake_configure_file",
+    "maintainers": [
+        {
+            "email": "daisuke.nishimatsu1021@gmail.com",
+            "name": "Daisuke Nishimatsu"
```

Comment:
Could you add your GitHub handle? That will allow the bot to mention you when something needs to be done (e.g. a new version is submitted).

Reply:
added https://github.com/bazelbuild/bazel-central-registry/pull/1454/commits/fea144cd758e31bf2787e1485ffd0d886938ce03